src/modules/lists/lists.controller.ts

```diff
@@ -70,4 +71,49 @@ export class ListsController {
     };
   }
 
+  @ApiOperation({ title: 'Updates a given list' })
+  @Put('/:listid')
+  async updateList(@Req() request: Request, @Param('userid') userId: string, @Param('listid') listId: string, @Body() data: UpdateList) {
+    const current: User = await this.usersService.findByAuth0Id(request.user.auth0Id);
+    const user: User = await this.usersService.findById(userId);
+
+    // check if name is provided
+    if (!data.name) {
```

Comment:
We already have this validation [in the pipes](https://github.com/Coding-Coach/find-a-mentor-api/blob/master/src/modules/lists/dto/list.dto.ts#L12), please use  the`ValidationPipe` like this: https://github.com/Coding-Coach/find-a-mentor-api/blob/master/src/modules/users/users.controller.ts#L125